### DISCUSSION TITLE
Removing external link styling

### DISCUSSION THREAD

**Discussion by aditvishnu** [luizdepra/hugo-coder#781] [Q&A] [answered]
Is there an option to remove the new external link styling that was added in #779 ? 
Thanks!

**Comment by luizdepra** [comment, marked as answer]
Hey!

The simplest way is overriding [this styling rule](https://github.com/luizdepra/hugo-coder/blob/main/assets/scss/_content.scss#L23-L26) in a custom CSS file.
